src/main/java/org/sciborgs1155/robot/drive/SparkModule.java

```diff
@@ -61,6 +76,12 @@ public SparkModule(int drivePort, int turnPort, Rotation2d angularOffset) {
 
     turnMotor = new CANSparkMax(turnPort, MotorType.kBrushless);
     turningEncoder = turnMotor.getAbsoluteEncoder();
+    turnPID = turnMotor.getPIDController();
+
+    // TODO: Re-tune
+    turnPID.setP(Turning.PID.SPARK.P);
+    turnPID.setI(Turning.PID.SPARK.I);
+    turnPID.setD(Turning.PID.SPARK.D);
```

Comment:
You're missing some turning configurations; continuous wrapping should be enabled and properly configured (min and max) for the turn motor.